### ISSUE TITLE
python core dumped when the matrix file is big

### ISSUE THREAD

**Issue by ZihuaLiu666** [kcakdemir/HiCPlotter#25]
Hi! I really like the HiCPlotter scripts! However, when I plotted the whole genome Hi-C interaction heat map with 400 Mb iced matrix file, it seems that HiCPlotter couldn't work very well. The broken message is: 
**line 35: 71609 Segmentation fault      (core dumped)**
Could you please fix it for me? Thank you very much in advance!

**Comment by kedarrudrawar** [comment]
Hi, I'm also getting Segmentation fault error when I plot Hi-C interactions with 20 kb iced matrix file. However, it works on 40 kb files, so any input would be appreciated! 
Thanks!

**Comment by kcakdemir** [comment]
Hi folks,
I think the issue is computer's memory. I havent had any issues with 20kb matrices (or 400 Mb matrix files) before for plotting the individual chromosome. Are you trying to plot whole genome?

**Comment by ZihuaLiu666** [comment]
@kcakdemir Yup, it should be the matter of computer's memory... It's a pushover for HiCPlotter to plot single chromosome Hi-C heatmap!

**Comment by kcakdemir** [comment]
My personal opinion is that plotting a whole chromosome (or genome) with a lower resolution would not be veru beneficial as the bin numbers grow and it's hard to see any additional information in a big matrix.

**Comment by ZihuaLiu666** [comment]
Hi! Plotting a whole genome with a lower resolution file is a smart idea! I will try it! Thanks for your generous help. BTW, HiCPlotter is super neat!
